lib/preprocessor.js

```diff
@@ -60,6 +62,15 @@ var createCoveragePreprocessor = function(logger, basePath, reporters, coverageR
       // remember the actual immediate instrumented JS for given original path
       sourceCache[jsPath] = content;
 
+      if (includeAllSources) {
```

Comment:
Ah rebase, gotcha. I have all the requested changes in now (unless I forgot something). Did you want me to try a rebase now?